aws/iac/master-compliance-test.json

```diff
@@ -2575,6 +2575,58 @@
                             "service": []
                         }
                     ]
+                },
+                {
+                    "masterTestId": "TEST_SECRETS_1",
+                    "masterSnapshotId": [
+                        "CFR_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(secrets.rego)",
+                    "evals": [
+                        {
+                            "id":"PR-AWS-0028-RGX",
+                            "eval": "data.rule.gl_aws_secrets",
+                            "message": "data.rule.gl_aws_secrets_err",
+                            "remediationDescription":"",
+                            "remediationFunction":""
+                        }
+                    ],
+                    "title": "",
+                    "description": "",
```

Comment:
Description: `We should not have any secret in the source code. Make sure to put the secrets in a vault`